packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -44,7 +43,7 @@ const CustomModalContent = styled(Modal.Content)`
   max-height: 100%;
 `;
 
-const RelationModal = ({ open, onToggle, id, model, relationUrl }: RelationModalProps) => {
+const RelationModal = ({ open, onToggle, id }: RelationModalProps) => {
```

Comment:
We don't need id either, I removed it over on https://github.com/strapi/strapi/pull/22998/files#diff-8cdbba82c84d96b805f8a065b7c2348b77f96b0f09e0e4cd8933a6f8c0fbc5df